Makefile

```diff
@@ -79,6 +79,10 @@ endif
 prometheus-init:
 	cd ansible && ANSIBLE_SSH_RETRIES=$(ANSIBLE_SSH_RETRIES) ansible-playbook -i hosts  -u root prometheus.yaml -f 10
 
+.PHONY: loadrunners-init
+loadrunners-init:
```

Comment:
Do we need to add this as a step in the README? (or as part of an existing step)

Or maybe you added in `method.md` for the 200-node test? (actually, I think I prefer this option because it leaves the README in this repo more general-purpose)

Reply:
The second. I updated `method.md` in https://github.com/cometbft/cometbft/pull/878.